pages/quiz/[question].js

```diff
@@ -239,25 +237,19 @@ const NavBar = styled.footer`
 const StyledQuizButton = styled.button`
   font-family: Georgia, 'Times New Roman', Times, serif;
   font-size: 42px;
-  padding-bottom: 9px;
+  padding-bottom: 7px;
   color: #8a98a5;
   border: none;
   background-color: rgba(31, 31, 35, 1);
-  :hover {
-    border-bottom: 3px solid white;
-    margin-top: 1px;
-  }
+  border-bottom: 5px solid white;
+  margin-top: 2px;
 `;
 const StyledWineGlasButton = styled.button`
   font-family: Georgia, 'Times New Roman', Times, serif;
```

Comment:
I'm not a huge fan of declaring the font inside of a styled component. Such things should preferably be handled by your GlobalStyles. You can define variables for your fonts as well, if you need them.